src/Entity/Hashtag.php

```diff
@@ -5,28 +5,37 @@
 class Hashtag extends AbstractEntity
 {
     /**
-     * {@inhertDoc}.
```

Comment:
How come you're explicitly settings the return type here and not relying on `{@inheritdoc}`?